pkg/auth/authTokensGet.go

```diff
@@ -19,29 +20,39 @@ import (
 func GetTokens(
 	apiClient *galasaapi.APIClient,
 	console spi.Console,
+	loginId string,
 ) error {
 
-	authTokens, err := getAuthTokensFromRestApi(apiClient)
+	authTokens, err := getAuthTokensFromRestApi(apiClient, loginId)
 
 	if err == nil {
-		summaryFormatter := tokensformatter.NewTokenSummaryFormatter()
-
-		var outputText string
-		outputText, err = summaryFormatter.FormatTokens(authTokens)
-
-		if err == nil {
-			console.WriteString(outputText)
-		}
+		err = formatFetchedTokensAndWriteToConsole(authTokens, console)
 	}
 
 	return err
 }
 
-func getAuthTokensFromRestApi(apiClient *galasaapi.APIClient) ([]galasaapi.AuthToken, error) {
+func getAuthTokensFromRestApi(apiClient *galasaapi.APIClient, loginId string) ([]galasaapi.AuthToken, error) {
 	var context context.Context = nil
 	var authTokens []galasaapi.AuthToken
+	var err error
+
+	apiCall := apiClient.AuthenticationAPIApi.GetTokens(context)
 
-	tokens, resp, err := apiClient.AuthenticationAPIApi.GetTokens(context).Execute()
+	if loginId != "" {
+
+		loginId, err = validateLoginIdFlag(loginId)
+
+		if err == nil {
+			apiCall = apiCall.LoginId(loginId)
+		}
+	}
+
+	if err != nil {
+		return authTokens, err
+	}
```

Comment:
To maintain consistency with the rest of the codebase, avoid multiple returns in the same function